source/MRViewer/MRViewer.h

```diff
@@ -628,6 +643,8 @@ class Viewer
 
     std::shared_ptr<HistoryStore> globalHistoryStore_;
 
+    std::shared_ptr<SpaceMouseHandler> spaceMouseHandler_;
```

Comment:
as far as it is purely private, no need in shared_ptr
```suggestion
    std::unique_ptr<SpaceMouseHandler> spaceMouseHandler_;

Reply:
fixed

Reply:
fixed